projects/fusion-ui/components/table/v3/components/table-basic/table-basic.component.ts

```diff
@@ -135,6 +159,46 @@ export class TableBasicComponent implements OnInit, OnDestroy {
         return isLast && (hasMore || this.loadingChildRows[rowIndex] || this.failedChildRows[rowIndex]);
     }
 
+    getRowspanIndexes(row): number[] {
+        return [...Array(this.tableService.getMaxRowspanInColumn(row)).keys()].filter(Boolean);
+    }
+
+    private setHoverForRowspan() {
+        const rowElements = this.elementRef.nativeElement.querySelectorAll('tr[data-row-idx]');
+        const events = ['mouseenter', 'mouseleave'];
```

Comment:
do subscription for 2 events 'mouseenter' and  'mouseleave' for element <tr> with attribute "data-row-idx".